benchmark/buffers/dataview-set.js

```diff
@@ -41,17 +41,19 @@ function main(conf) {
 
 function benchInt(dv, fn, len, le) {
   var m = mod[fn];
+  var method = dv[fn];
```

Comment:
Um... well, maybe, but in that case makes sense to change other places in this file as well. And, generally, unless already consistent with a file styling, I'm personally not a big fan of using block-scoped variables in benchmark code at least until V8 fixes their deopts (it does affect numbers quite a bit at the moment), but I can change if you really wish.